ygnmi/ygnmi.go

```diff
@@ -112,6 +112,19 @@ func (v *Value[T]) IsPresent() bool {
 	return v.present
 }
 
+// String returns a user-readable string for the value.
+func (v *Value[T]) String() string {
+	path, err := ygot.PathToString(v.Path)
+	if err != nil {
+		path = v.Path.String()
+	}
+	val := fmt.Sprintf("%+v", v.val)
```

Comment:
nit: invert so the conditional has a positive polarity?

val := "(not present)"
if v.present {
  val := fmt.Sprintf("%+v", v.val)
}